docs/docs/60-new-docs/40-operator-guide/10-basic-installation.md

```diff
@@ -75,7 +81,7 @@ These instructions were tested with:
     The above commands will leave you with values assigned to `$hashed_pass` and
     `$signing_key`. These will be used in the next step.
 
-1. Install Kargo with default configuration and your chosen admin account
+2. Install Kargo with default configuration and your chosen admin account
```

Comment:
We number lists with 1. so that we don't have to renumber when items are removed or added.